test/integration/targets/win_psmodule/tasks/main.yml

```diff
@@ -16,112 +16,13 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-- name: get powershell version
-  win_shell: $PSVersionTable.PSVersion.Major
-  register: version
 
-- name: install module from Powershell Gallery
-  win_psmodule:
-    name: "{{ powershell_module }}"
-    state: present
-  register: module_setup
-  when: version.stdout_lines[0]|int >= 5
+- name: get facts
+  setup:
```

Comment:
The current `ansible-test` implementation does not automatically gather facts for Windows integration tests.